src/pages/scientificServices/pipelines/components/inputs/PipelineFileInput.tsx

```diff
@@ -186,11 +186,7 @@ export const PipelineFileInput: React.FC<PipelineInputSelectorProps> = ({
                         }}
                         aria-label='Remove selected file'
                       >
-                        <Icon
-                          icon='times'
-                          size={24}
-                          color={uploadState?.progress === 100 ? colors.disabled() : '#4D72AA'}
-                        />
+                        <Icon icon='times' size={24} color={uploadState?.progress ? colors.disabled() : '#4D72AA'} />
```

Comment:
these changes are unrelated to the tickets, but i noticed this logic wasn't quite right and wanted to update it so the clear file button is disabled during an upload, rather than just after an upload completes.

a user shouldn't ever see this anyway because we show them the progress bar after initiating an upload, but it still bugged me